src/routers/alpha-router/alpha-router.ts

```diff
@@ -929,92 +912,34 @@ export class AlphaRouter
           routingConfig
         )
       );
-      /// @dev only add mixedRoutes in the case where no protocols were specified, and if TradeType is correct
-      if (
-        tradeType === TradeType.EXACT_INPUT &&
-        (this.chainId === ChainId.MAINNET || this.chainId === ChainId.GÖRLI) &&
-        /// The cases where protocols = [] and protocols = [V2, V3, MIXED]
-        (protocolsSet.size == 0 || protocolsSet.has(Protocol.MIXED))
-      ) {
-        log.info(
-          { protocols, swapType: tradeType },
-          'Routing across MixedRoutes'
-        );
-        quotePromises.push(
-          this.getMixedRouteQuotes(
-            tokenIn,
-            tokenOut,
-            amounts,
-            percents,
-            quoteToken,
-            mixedRouteGasModel,
-            tradeType,
-            routingConfig
-          )
-        );
-      }
-    } else {
-      if (
-        protocolsSet.has(Protocol.V3) ||
-        (protocolsSet.size == 0 && !V2_SUPPORTED.includes(this.chainId))
-      ) {
-        log.info({ protocols, swapType: tradeType }, 'Routing across V3');
-        quotePromises.push(
-          this.getV3Quotes(
-            tokenIn,
-            tokenOut,
-            amounts,
-            percents,
-            quoteToken,
-            v3gasModel,
-            tradeType,
-            routingConfig
-          )
-        );
-      }
-      if (protocolsSet.has(Protocol.V2)) {
-        log.info({ protocols, swapType: tradeType }, 'Routing across V2');
-        quotePromises.push(
-          this.getV2Quotes(
-            tokenIn,
-            tokenOut,
-            amounts,
-            percents,
-            quoteToken,
-            gasPriceWei,
-            tradeType,
-            routingConfig
-          )
-        );
-      }
-      /// If protocolsSet is not empty, and we specify mixedRoutes, consider them if the chain has v2 liq
-      /// and tradeType === EXACT_INPUT
-      if (
-        protocolsSet.has(Protocol.MIXED) &&
-        (this.chainId === ChainId.MAINNET || this.chainId === ChainId.GÖRLI) &&
-        tradeType == TradeType.EXACT_INPUT
-      ) {
-        log.info(
-          { protocols, swapType: tradeType },
-          'Routing across MixedRoutes'
-        );
-        quotePromises.push(
-          this.getMixedRouteQuotes(
-            tokenIn,
-            tokenOut,
-            amounts,
-            percents,
-            quoteToken,
-            mixedRouteGasModel,
-            tradeType,
-            routingConfig
-          )
-        );
-      }
+    }
+
+    // Maybe Quote mixed routes
+    // if MixedProtocol is specified or no protocol is specified and v2 is supported AND tradeType is ExactIn
+    // AND is Mainnet or Gorli
+    const shouldQueryMixedProtocol = mixedProtocolSpecified || (noProtocolsSpecified && v2SupportedInChain)
+    if (shouldQueryMixedProtocol && tradeTypeIsExactInput && (isMainnet || isGorli)) {
+      log.info(
```

Comment:
It would be great if we added a linting/code prettifier task to this repo, my VSCode formatted this code and made some unwanted changes, and I'm not sure how to disable the auto-formatting locally 😅